packages/astro/package.json

```diff
@@ -64,7 +64,7 @@
     "@sentry/vite-plugin": "^2.22.6"
   },
   "devDependencies": {
-    "astro": "^3.5.0",
+    "astro": "^4.16.8",
```

Comment:
We still support Astro 3.0.0 and upwards (https://docs.sentry.io/platforms/javascript/guides/astro/#prerequisites). I guess the build is failing because some types changed in v4?

Reply:
I updated this to reflect v4.

I'd rather the package stays up-to-date with the latest version, and we use e2e tests to test against older versions when needed. What do you think?

Reply:
that makes sense to me IMHO!